components/TextFields/src/MDCTextInputController.m

```diff
@@ -421,15 +430,16 @@ - (void)movePlaceholderToUp:(BOOL)isToUp {
     animationBlock = ^{
       self.textInput.placeholderLabel.transform = floatingPlaceholderScaleTransform;
 
-      self.textInput.placeholderLabel.textColor = self.textInput.tintColor;
+      self.textInput.placeholderLabel.textColor =
+      self.isDisplayingErrorText ? self.errorColor : self.floatingPlaceholderColor;
```

Comment:
Done. Also this is an out of date snapshot. This line was changed logically also.